public/components/overview/office-panel/views/office-stats.tsx

```diff
@@ -11,40 +11,49 @@
  * Find more information about this on the LICENSE file.
  */
 
-import { EuiDescriptionList, EuiFlexItem, EuiFlexGroup, EuiTitle, EuiCallOut } from '@elastic/eui';
-import moduleLogo from '../../../../assets/office365.svg';
 import React from 'react';
-import './office-stats.scss';
+import { EuiDescriptionList, EuiPanel } from '@elastic/eui';
+import { PanelModuleConfiguration } from '../../../common/modules/panel';
+import moduleLogo from '../../../../assets/office365.svg';
 
-export const OfficeStats = ({ listItems = [] }) => {
-  const logoStyle = { width: 30 };
-  return (
-    <EuiFlexGroup direction={'column'} alignItems={'flexStart'}>
-      <EuiFlexItem>
-        <EuiFlexGroup>
-          <EuiFlexItem className={'wz-justify-center'}>
-            <img alt="moduleLogo" src={moduleLogo} style={logoStyle} />
-          </EuiFlexItem>
-          <EuiFlexItem>
-            <EuiTitle size={'s'}>
-              <h4 className={'office-stats-title'}>Office 365</h4>
-            </EuiTitle>
-            <EuiTitle size={'xs'}>
-              <h5 className={'office-stats-subtitle'}>Module configuration</h5>
-            </EuiTitle>
-          </EuiFlexItem>
-        </EuiFlexGroup>
-      </EuiFlexItem>
-      <EuiFlexItem>
-        {
-          listItems.length ? (
-            <EuiDescriptionList className={'office-description-list'} listItems={listItems} compressed />) : (
-            <EuiCallOut className={'office-stats-callout-warning'}
-              title="Module configuration unavailable"
-              color="warning"
-              iconType="warning" />)
-        }
-      </EuiFlexItem>
-    </EuiFlexGroup>
-  );
+const settings = [
+  { field: 'enabled', label: 'Enabled'},
+  { field: 'only_future_events', label: 'Collect events generated since Wazuh agent was started'},
+  { field: 'curl_max_size', label: 'Maximum size allowed for the Office 365 API response'},
+  { field: 'interval', label: 'Interval between Office 365 wodle executions in seconds'},
+  { field: 'api_auth', label: 'Credentials', render: (value) => value.map(v => 
+    <EuiPanel paddingSize='s' key={`module_configuration_api_auth_${v.tenant_id}_${v.client_id}`}>
+      <EuiDescriptionList listItems={[
+        {title: 'Tenant ID', description: v.tenant_id},
+        {title: 'Client ID', description: v.client_id},
+        {title: 'Client secret', description: v.client_secret},
+        {title: 'Path file of client secret', description: v.client_secret_path},
+      ].filter(item => item.description !== undefined)}/>
```

Comment:
It would be interesting if we manage undefined with the same form in all occurrences

Reply:
**fix**: Replaced as compare against `undefined`. Used `typeof`